src/Sql/Sql/help/Remove-AzSqlInstanceServerTrustCertificate.md

```diff
@@ -0,0 +1,299 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
+schema: 2.0.0
+---
+
+# Remove-AzSqlInstanceServerTrustCertificate
+
+## SYNOPSIS
+Removes an Azure SQL Server Trust Certificate.
+
+## SYNTAX
+
+### DeleteByNameParameterSet (Default)
+```
+Remove-AzSqlInstanceServerTrustCertificate [-ResourceGroupName] <String> [-InstanceName] <String>
+ [-CertificateName] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### DeleteByParentObjectParameterSet
+```
+Remove-AzSqlInstanceServerTrustCertificate [-CertificateName] <String>
+ [-Instance] <AzureSqlManagedInstanceModel> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### DeleteByInputObjectParameterSet
+```
+Remove-AzSqlInstanceServerTrustCertificate [-Certificate] <AzureSqlInstanceServerTrustCertificateModel>
```

Comment:
Certificate -> InputObject Instance->InstanceObject, please refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/piping-best-practices.md#using-the-inputobject-parameter

Reply:
thanks, my bad, didn't understand from the doc that those should literally be the parameters names (thought they were just generic placeholders)